### ISSUE TITLE
"No expiry configured." when AUTO is configured

### ISSUE THREAD

**Issue by cdamken** [owncloud/core#28509]
Ref: https://github.com/owncloud/enterprise/issues/2148

The message from `occ versions:expire` and `occ trashbin:expire` is confusing the `auto` is configured:

```
root@snoopy3:/var/www/owncloud/config# occ versions:expire
No expiry configured.
root@snoopy3:/var/www/owncloud/config# occ trashbin:expire
No expiry configured.
```

Looking the documentation:
old versions should be erase if more than 50% of the quota is used
https://doc.owncloud.org/server/9.1/admin_manual/configuration_files/file_versioning.html
and  Trashbin files are older than 30 days
https://doc.owncloud.org/server/9.1/admin_manual/configuration_files/trashbin_options.html?highlight=trashbin

Those procedures are handle when files are uploaded when auto is configured and is not necessary to run the occ command in that configuration.

IMO instead `No expiry configured.` (where the admin thinks that he forgot to configure something)  the message should be: `Auto expiration is configured - expiration will be handle automatically.`

@tomneedham @VicDeo @DeepDiver1975  Your opinion please.

**Comment by cdamken** [comment]
@tomneedham @VicDeo @DeepDiver1975 @pmaier1 @hodyroff Your opinion, please.

**Comment by hodyroff** [comment]
Makes sense, just make a PR ... when its auto we should not say that no expiry is configured. And the auto values must be in the documentation.

**Comment by PVince81** [comment]
Likely not critical enough for 9.1, so 10.0.5 only. Let me know if you think otherwise.

**Comment by VicDeo** [comment]
to be released with 10.0.5

**Comment by VicDeo** [comment]
Closing as fixed
